apps/web-mzima-client/src/app/shared/components/onboarding/onboarding.component.scss

```diff
@@ -49,12 +49,12 @@
 
   &__head {
     display: flex;
+    align-items: center;
+    justify-content: center;
     flex-wrap: wrap;
     position: relative;
     margin: 0 -4px 24px;
```

Comment:
Hello @Felysilver 
This fix you sent is different from and does not address what you were asked to do in the issue. 

Steps to reproduce the issue you were asked to fix:
- Go to /map
- Change your language to arabic
- In your dev tool, click on "application" tab, go to "storage" and clear site data
- Reload the page (while still in language Arabic), you will see the onboarding modal popup
- Keep going to the next modal until you get to "sorting". The overlay is there but the modal is no longer there.

Reply:
So here's what you can do @Felysilver 
1. You can create another github issue for this particular fix you are submitting in this PR, then link that new issue you created to this PR - I will then review it.
2. For my [comment above here](https://github.com/ushahidi/platform-client-mzima/pull/821#discussion_r1532243425), would you like to also fix this issue - now that I have shared how to reproduce it?